engine/components/sql/src/main/resources/META-INF/services/pl.touk.nussknacker.engine.api.component.ComponentProvider

```diff
@@ -1 +1,2 @@
-pl.touk.nussknacker.sql.DatabaseEnricherComponentProvider
\ No newline at end of file
+pl.touk.nussknacker.sql.DatabaseEnricherComponentProvider
+pl.touk.nussknacker.sql.db.ignite.IgniteEnricherComponentProvider
```

Comment:
Do we actually need new class here? Maybe some "auto-dialect-detection" instead?

Reply:
auto-metadprovider detection was described in proposal - @kdutkowski wrote a message on rocket that he used this approach to simplify implemention. IMO this way is a little bit better then using service loader because we don't have two levels of service loading. Or mabye you have in mind some other approach @mproch ?

Reply:
What I understand as 'auto-detection' is a single ComponentProvider that can instantiate the right DbMetaDataProvider  based on something like driver class